<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Admin-PC" name="com.parameterization.TestParameters" tests="2" failures="0" timestamp="8 Jul 2020 19:41:34 GMT" time="0.003" errors="1">
  <testcase name="testCaseTwo" time="0.002" classname="com.parameterization.TestParameters"/>
  <testcase name="testCaseOne" time="0.001" classname="com.parameterization.TestParameters">
    <error type="org.testng.TestException" message="
Method TestParameters.testCaseOne(java.lang.String)[pri:0, instance:com.parameterization.TestParameters@5ae9a829] should have thrown an exception of type class java.lang.NumberFormatException">
      <![CDATA[org.testng.TestException: 
Method TestParameters.testCaseOne(java.lang.String)[pri:0, instance:com.parameterization.TestParameters@5ae9a829] should have thrown an exception of type class java.lang.NumberFormatException
	at org.testng.internal.Invoker.handleInvocationResults(Invoker.java:1426)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:701)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- testCaseOne -->
</testsuite> <!-- com.parameterization.TestParameters -->
